etl/src/main/java/org/pih/hivmigration/etl/sql/VisitMigrator.groovy

```diff
@@ -0,0 +1,42 @@
+package org.pih.hivmigration.etl.sql
+
+class VisitMigrator extends SqlMigrator {
+
+    void migrate() {
+
+        executeMysql("Create visits from existing encounters", '''
+            -- Get visit type "Clinic or Hospital Visit"
+            SET @visit_type_id = (SELECT visit_type_id FROM visit_type WHERE uuid = 'f01c54cb-2225-471a-9cd5-d348552c337c');
+            SET @encounter_type_exclusions = '';  -- a comma-separated string like '1,2,3'
+            
+            INSERT INTO visit
+            (patient_id,   visit_type_id,  date_started,   date_stopped,   location_id,   creator,   date_created, voided, uuid)
+            SELECT  e.patient_id, @visit_type_id, e.date_started, e.date_stopped, e.location_id, e.creator, now(),     0, uuid()
+            FROM
+                (
+                    SELECT patient_id,
+                           Subtime(Min(encounter_datetime), '00:05:00') AS date_started,  /* visit must start before first enc */
+                           Addtime(Max(encounter_datetime), '00:05:00') AS date_stopped,  /* visit must end after last enc */
```

Comment:
Per our discussion today, based on how we are doing retrospective visits in the PIH EMR, the start date should have a time component of "00:00:00" and the end date should have one of "23:59:59"